modules/EvseManager/EvseManager.cpp

```diff
@@ -167,52 +167,43 @@ void EvseManager::ready() {
 
         auto sae_mode = types::iso15118_charger::SAE_J2847_Bidi_Mode::None;
 
-        types::iso15118_charger::SetupPhysicalValues setup_physical_values;
-
         // Set up energy transfer modes for HLC. For now we only support either DC or AC, not both at the same time.
         std::vector<types::iso15118_charger::EnergyTransferMode> transfer_modes;
         if (config.charge_mode == "AC") {
+            types::iso15118_charger::SetupPhysicalValues setup_physical_values;
             setup_physical_values.ac_nominal_voltage = config.ac_nominal_voltage;
+            r_hlc[0]->call_set_physical_values(setup_physical_values);
 
             transfer_modes.push_back(types::iso15118_charger::EnergyTransferMode::AC_single_phase_core);
             if (config.three_phases) {
                 transfer_modes.push_back(types::iso15118_charger::EnergyTransferMode::AC_three_phase_core);
             }
 
         } else if (config.charge_mode == "DC") {
-            // transfer_modes.push_back(types::iso15118_charger::EnergyTransferMode::DC_core);
             transfer_modes.push_back(types::iso15118_charger::EnergyTransferMode::DC_extended);
 
-            powersupply_capabilities = r_powersupply_DC[0]->call_getCapabilities();
-
-            // subscribe to run time updates e.g. due to de-rating
-            r_powersupply_DC[0]->subscribe_update_capabilities(
-                [this](auto caps) { update_powersupply_capabilities(caps); });
-
-            updateLocalMaxWattLimit(powersupply_capabilities.max_export_power_W);
-
-            setup_physical_values.dc_current_regulation_tolerance =
-                powersupply_capabilities.current_regulation_tolerance_A;
-            setup_physical_values.dc_peak_current_ripple = powersupply_capabilities.peak_current_ripple_A;
-
+            // Init power supply capabilities with safe defaults
+            types::power_supply_DC::Capabilities psu_caps;
+            psu_caps.bidirectional = false;
+            psu_caps.peak_current_ripple_A = 0.5;
+            psu_caps.current_regulation_tolerance_A = 0.5;
+            psu_caps.max_export_current_A = 0;
+            psu_caps.max_export_power_W = 0;
+            psu_caps.max_export_voltage_V = 60;
+            psu_caps.min_export_voltage_V = 0;
+            psu_caps.min_export_current_A = 0;
```

Comment:
if this would be wrapped into a helper function called `types::power_supply_DC::Capabilities get_sane_default_power_supply_capabilities()` or similar, the comment wouldn't be necessary and probably easier to read.